plugins/sentry/samson_sentry.gemspec

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+Gem::Specification.new 'samson_sentry', '0.0.0' do |s|
+  s.summary = 'Samson Sentry plugin'
+  s.authors = ['zendesk-mattlefevre']
+  s.email = ['matthew.lefevre@zendesk.com']
+  s.files = Dir['{app,config,db,lib}/**/*']
+
+  s.add_runtime_dependency 'sentry-rails'
+  s.add_runtime_dependency 'sentry-ruby'
```

Comment:
don't need this line afaik

Reply:
You may be right; just following [what's in the docs](https://docs.sentry.io/platforms/ruby/guides/rails/).